examples/server-kitchen-sink/client/storybook.js

```diff
@@ -0,0 +1,19 @@
+import camelcase from 'camelcase';
+import { configure } from '@storybook/server';
+
+import stories from './storybook.stories';
+
+const port = process.env.PORT || 8080;
+
+const fetchHtml = async (id, params) => {
+  const [component, story] = id.split('--').map(s => camelcase(s));
+
+  const url = new URL(`http://localhost:${port}/storybook_preview/${component}/${story}`);
+  url.search = new URLSearchParams(params).toString();
+
+  // eslint-disable-next-line no-undef
+  const response = await fetch(url);
```

Comment:
What's the right way to fix this without the linting hack?

Reply:
https://github.com/eslint/eslint/issues/4015#issuecomment-334929169

Reply:
nice that was an easy fix